boards/samr21-xpro/include/board.h

```diff
@@ -48,6 +48,13 @@ extern "C" {
 #define AT86RF231_SLEEP    GPIO_7
 
 #define AT86RF231_SPI_SPEED SPI_SPEED_1MHZ
+
+/*Thread optimization for samr21*/
+#define RADIO_STACK_SIZE                (1024)
+#define RPL_PROCESS_STACKSIZE           (1024)
+
+/*samr21 is very memory constrained and does not run with the default RPL_MAX_ROUTING_ENTRIES (= 128)*/
+#define RPL_MAX_ROUTING_ENTRIES            (8)
```

Comment:
will this line interfere with the `RPL_MAX_ROUTING_ENTRIES` specified by command line (at compilation time) ? Which one has the higher priority?